package.json

```diff
@@ -1,5 +1,12 @@
 {
   "private": true,
+  "dependencies": {
+    "connect-history-api-fallback": "^1.2.0",
+    "gulp": "^3.9.1",
+    "gulp-load-plugins": "^1.2.2",
+    "proxy-middleware": "^0.15.0",
+    "url": "^0.11.0"
```

Comment:
It is, as are all of the others. I added a `dependencies` section, so that a build can be run on a clean environment that does not contain any development tools.